index.html

```diff
@@ -26,9 +26,6 @@ <h3 class="subheading">Working to promote transparency and empower citizens thro
   </div>
 </div>
 
-div> <!-- /container -->
-
-iv class="section2">
 <div class="container">
```

Comment:
nice catch! wonder if those should be there but got chopped up? @mikedillion you remember this in the jekyllization